clang/lib/DPCT/APINamesStream.inc

```diff
@@ -52,13 +52,3 @@ ASSIGNABLE_FACTORY(CONDITIONAL_FACTORY_ENTRY(
                             Diagnostics::TRY_EXPERIMENTAL_FEATURE,
                             ARG("cudaStreamEndCapture"),
                             ARG("--use-experimental-features=graph"))))
-
-ASSIGNABLE_FACTORY(CONDITIONAL_FACTORY_ENTRY(
```

Comment:
I wonder why we cannot use the rewriter inc file to handle this API migration? May be you can try to add the `cudaStreamLegacy` mapping in EA (and MapName.cpp) then the `ARG(0)` can be migrated properly.

Reply:
Hi @tomflinda , I saw your migration logic for `cudaStreamQuery` this way and added the same. Is it possible to add `cudaStreamLegacy/cudaStreamDefault` in `MapNames.cpp` and doing EA rather than doing it in AST?

Reply:
@daiyaan-ahmed6  
Before the EA framework is implemented, all the CUDA APIs are implemented in AST, which has limitations. That's how EA is introduced. So, using the EA framework is the first choice for implementing the migration logic for new API support.

Reply:
Can be considered as a new design for next release